src/features/partnerNetwork/types.ts

```diff
@@ -162,12 +187,93 @@ export interface PartnerNetworkInitialState {
 }
 
 export interface PartnerNetworkDataGrid {
-  bpn: string
-  name: string
-  legalForm: string
-  country: string
-  street: string
-  zipCode: string
-  city: string
+  bpnl: string
+  legalName: string
+  legalForm: any
+  legalAddress: any
```

Comment:
define and use types or interfaces

Reply:
As discussed, this is needed for unit test case. but have removed any from all other places